tasks/systemd.yml

```diff
@@ -0,0 +1,47 @@
+- name: Make sure systemd unit file directory does exist
+  file:
+    path: /lib/systemd/system/
```

Comment:
That's wrong. Administrator-created units go to `/etc/systemd/system`, this directory always exists

Reply:
Is this in fact a administrator created unit?

Reply:
Yes, you are the administrator, not the distribution vendor